shell/platform/android/io/flutter/plugin/platform/PlatformViewWrapper.java

```diff
@@ -132,61 +76,7 @@ public void setTouchProcessor(@Nullable AndroidTouchProcessor newTouchProcessor)
     touchProcessor = newTouchProcessor;
   }
 
-  /**
-   * Sets the texture where the view is projected onto.
-   *
-   * <p>{@link PlatformViewWrapper} doesn't take ownership of the {@link SurfaceTexture}. As a
-   * result, the caller is responsible for releasing the texture.
-   *
-   * <p>{@link io.flutter.view.TextureRegistry} is responsible for creating and registering textures
-   * in the engine. Therefore, the engine is responsible for also releasing the texture.
-   *
-   * @param newTx The texture where the view is projected onto.
-   */
-  @SuppressLint("NewApi")
-  public void setTexture(@Nullable SurfaceTexture newTx) {
-    if (Build.VERSION.SDK_INT < 23) {
-      Log.e(
-          TAG,
-          "Platform views cannot be displayed below API level 23. "
-              + "You can prevent this issue by setting `minSdkVersion: 23` in build.gradle.");
-      return;
-    }
-
-    tx = newTx;
-
-    if (bufferWidth > 0 && bufferHeight > 0) {
-      tx.setDefaultBufferSize(bufferWidth, bufferHeight);
-    }
-
-    if (surface != null) {
-      surface.release();
-    }
-    surface = createSurface(newTx);
-
-    // Fill the entire canvas with a transparent color.
-    // As a result, the background color of the platform view container is displayed
-    // to the user until the platform view draws its first frame.
-    final Canvas canvas = surface.lockHardwareCanvas();
-    try {
-      canvas.drawColor(Color.TRANSPARENT, PorterDuff.Mode.CLEAR);
-      onFrameProduced();
-    } finally {
-      surface.unlockCanvasAndPost(canvas);
-    }
-  }
-
-  @NonNull
-  @VisibleForTesting
-  protected Surface createSurface(@NonNull SurfaceTexture tx) {
-    return new Surface(tx);
-  }
-
-  /** Returns the texture where the view is projected. */
-  @Nullable
-  public SurfaceTexture getTexture() {
-    return tx;
-  }
+  public void setRenderTarget(@Nullable PlatformViewRenderTarget renderTarget) {}
```

Comment:
Why is this a no-op? Shouldn't it change `this.renderTarget`?

Reply:
This was dead code. Removed.